index.js

```diff
@@ -13,5 +13,20 @@ module.exports = (robot, _, Settings = require('./lib/settings')) => {
     if (defaultBranch && settingsModified) {
       return Settings.sync(context.github, context.repo())
     }
+    else if (!defaultBranch && settingsModified) {
+      let settingChangeCommits = payload.commits.filter(commit => {  //get all commits that change settings.yml
```

Comment:
instead of using filter here use find() as it will return a single object and you wont have to iterate in array cause you will always get a single file

Reply:
@knrt10 We want to create status for all the commits in a push event that modifies the settings right? Why would there be only one commit in a push event?